docs/advanced_topics/customization/page_editing_interface.md

```diff
@@ -159,6 +159,60 @@ To unregister, call `unregister_image_format` with the string of the `name` of t
 Unregistering ``Format`` objects will cause errors when viewing or editing pages that reference them.
 ```
 
+(date_field_validation)=
+
+## Date Field Validation
+
+Wagtail provides some built-in validators for common date validation scenarios. These validators can be applied to `DateField` and `DateTimeField` instances to enforce specific constraints on date input.
```

Comment:
Given that `NoFutureDateValidator` is the only validator available right now, I think we can drop this paragraph.